build-packages.mjs

```diff
@@ -14,7 +14,11 @@ import glob from 'fast-glob';
 
 const exec = promisify(childProcess.exec);
 
-const rootDir = path.dirname(fileURLToPath(import.meta.url));
+// On Windows, `fileURLToPath()` returns a path with "back slashes" ("\").
+// But `fast-glob` doesn't find any files when `sourceDir` contains "back slashes"
+// (Windows) instead of "forward slashes" (Linux).
+// To fix that, convert all "back slashes" to "forward slashes".
+const rootDir = path.dirname(fileURLToPath(import.meta.url)).replaceAll('\\', '/');
```

Comment:
Maybe move this to line 33 for sourceDir? Likely path.join adds another backslash

Reply:
@WikiRik You're right:

```
@@@@@ rootDir c:/dev-github/sequelize
@@@@@ packageDir c:/dev-github/sequelize/packages/core
@@@@@ sourceDir c:\dev-github\sequelize\packages\core\src
```